### PR TITLE
[BACKLOG-41380] fix: show formatted value in tooltip for radar

### PR DESCRIPTION
**Pull request by AAravind27**

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 9m 20s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox
```
---
:ok_hand: All tests passed!
 
Tests run: **5241**, Failures: **0**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/common-ui/job/PR-1780/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: impl/client/src/main/javascript/web/pentaho/visual/views/echarts/Radar.js [hunk 1]
```diff
@@ -183,6 +189,14 @@ define([
           rowIndexes: rowIndexes
         };
 
+        var tooltipString = record.name !== "" ? ( record.name + "<br />" ) : "";
```
Comment (dcleao) [inline impl/client/src/main/javascript/web/pentaho/visual/views/echarts/Radar.js:192]: Shouldn't `record.name` be HTML escaped as well?

**Review by dcleao — CHANGES_REQUESTED** [review]

File: impl/client/src/main/javascript/web/pentaho/visual/views/echarts/Radar.js (hunk 1 above)
Comment (AAravind27) [reply impl/client/src/main/javascript/web/pentaho/visual/views/echarts/Radar.js]: It should be. I overlooked that detail.

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 10m 5s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox
```
---
:ok_hand: All tests passed!
 
Tests run: **5241**, Failures: **0**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/common-ui/job/PR-1780/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Pull request merged**